install.bash

```diff
@@ -7,52 +7,77 @@ if [[ "$(id -u)" -ne 0 ]]; then
   exit 1
 fi
 if [[ $1 == "sync" ]]; then
-  install -m 644 "$BASEDIR"/zsync.* /etc/systemd/system/
-  systemctl daemon-reload
-  systemctl enable --now zsync.timer
+  if [[ "$(grep -o '^ID=.*$' /etc/os-release | cut -d'=' -f2)" == "alpine" ]]; then
+    install -m 755 "$BASEDIR"/zsync.cron /etc/periodic/daily/zsync
+  else 
+    install -m 644 "$BASEDIR"/zsync.timer /etc/systemd/system/
+    install -m 644 "$BASEDIR"/zsync.service /etc/systemd/system/
+    systemctl daemon-reload
+    systemctl enable --now zsync.timer
+  fi
   echo "#####     zsync service is now installed     #####"
   exit 0
 fi
-if [[ $(systemctl is-active zram-config.service) == "active" ]]; then
+if [[ "$(systemctl is-active zram-config.service 2> /dev/null)" == "active" ]] || [[ "$(rc-service zram-config status 2> /dev/null | sed -ne 's/^ \* status: \(.*\)/\1/p')" == "started" ]]; then
   echo -e "ERROR: zram-config service is still running.\\nPlease run \"sudo ${BASEDIR}/update.bash\" to update zram-config instead."
   exit 1
 fi
-if [[ -f /usr/local/sbin/zram-config ]]; then
+if [[ -f /usr/local/sbin/zram-config || -f /usr/sbin/zram-config ]]; then
   echo -e "ERROR: zram-config is already installed.\\nPlease run \"sudo ${BASEDIR}/update.bash\" to update zram-config instead."
   exit 1
 fi
 
-if ! dpkg -s 'gcc' 'make' 'libc6-dev' &> /dev/null; then
+if ! dpkg -s 'gcc' 'make' 'libc6-dev' &> /dev/null && ! [[ "$(grep -o '^ID=.*$' /etc/os-release | cut -d'=' -f2)" == "alpine" ]]; then
   echo "Installing needed packages (gcc, make, libc6-dev)"
   apt-get install --yes gcc make libc6-dev || exit 1
 fi
+if ! [[ "$(apk info 2> /dev/null | grep -E '^(gcc|make|fts-dev)' | tr '\n' ' ')" == "gcc make fts-dev " ]]; then
+  echo "Installing needed packages (gcc, make, fts-dev)"
+  apk add gcc make fts-dev || exit 1
+fi
```

Comment:
Correct me if I'm wrong but I believe you need to add a is alpine check to this if as well others when this fails on Debian the `apk add` command will always attempt to run and throw an error.